src/test/regress/expected/multi_reference_table.out

```diff
@@ -1133,12 +1133,8 @@ FROM
 WHERE
 	colocated_table_test_2.value_4 = reference_table_test.value_4
 RETURNING value_1, value_2;
- value_1 | value_2 
----------+---------
-       1 |       1
-       2 |       2
-(2 rows)
-
+ERROR:  cannot perform distributed planning for the given modification
```

Comment:
Should we put that into error message instead ? This error does not say anything other than "this query has something bad in it"